src/modules/site-v2/templates/primary/map.html

```diff
@@ -165,11 +163,18 @@
 const OSM_URL = 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png';
 const OSM_ATTRIB = '&copy; <a href="https://openstreetmap.org/copyright">OpenStreetMap</a> contributors';
 
-const southWest = L.latLng(-90, -180);
-const northEast = L.latLng(90, 180);
+const southWest = L.latLng(-120, -240);
+const northEast = L.latLng(120, 240);
 const bounds = L.latLngBounds(southWest, northEast);
 
-const map = L.map('map', {minZoom:2, maxBounds: bounds, scrollWheelZoom: false, zoomControl: false, attributionControl: false,}).setView([20, -100], 2);
+const map = L.map('map', {renderer: L.canvas(), 
+                          minZoom:2, 
+                          maxZoom: 10, 
+                          maxBounds: bounds, 
+                          scrollWheelZoom: false, 
+                          zoomControl: false, 
+                          attributionControl: false})
```

Comment:
quick note -- we might be required to display the attributionControl (should show that we are using leaflet) for the map. If `attributionControl` is set to true, does it show "leaflet" anywhere in the map?